### PR TITLE
Enhancements

### PR DESCRIPTION
**Pull request by mayankamencherla**
- On pop up, the button is disabled
- On dismiss of pop up button is enabled

### CODE REVIEW

File: frontend/checkout.phtml [hunk 1]
```diff
@@ -81,14 +81,18 @@
 </form>
 
 <script>
-jQuery(document).ready(function( $ ){
-    function showModal(response) {
+jQuery(document).ready(function( $ )
```
Comment (captn3m0) [inline frontend/checkout.phtml]: 1. no space around parameters in signatures ^
2. put function opening braces on the same line

File: frontend/checkout.phtml [hunk 2]
```diff
@@ -135,9 +147,21 @@ jQuery(document).ready(function( $ ){
...
     {
-        createOrder();
+        document.getElementById("btn-razorpay").disabled = true;
     }
+
+
+    function addEvent(element, evnt, funct)
+    {
+        if (element.attachEvent)
+            return element.attachEvent('on'+evnt, funct);
```
Comment (captn3m0) [inline frontend/checkout.phtml]: 3. space around `+`

File: frontend/checkout.phtml (hunk 1 above)
Comment (mayankamencherla) [inline frontend/checkout.phtml]: Changed

File: frontend/checkout.phtml (hunk 2 above)
Comment (mayankamencherla) [inline frontend/checkout.phtml]: Changed

**Pull request merged**